src/main/java/cn/nukkit/exception/ChatColorCharOutOfBoundsException.java

```diff
@@ -0,0 +1,7 @@
+package cn.nukkit.exception;
+
+public class ChatColorCharOutOfBoundsException extends Exception {
+    public ChatColorCharOutOfBoundsException(String message) {
+
+    }
```

Comment:
This constructor is doing nothing with the `message` parameter, it should be calling `super(message)`.